CMSOdataset2018-3DsphrdInvCollHT1080-Bakker/isa/a_2018_3dsphinvasion_ht1080_collagen_spheroid_invasion_assay_microscopy_imaging-N2.txt

```diff
@@ -0,0 +1,9 @@
+Sample Name	Protocol REF	Parameter Value[imaging modality]	Term Source REF	Term Accession Number	Parameter Value[imaging sequence type]	Term Source REF	Term Accession Number	Parameter Value[observation period]	Term Source REF	Term Accession Number	Unit	Term Source REF	Term Accession Number	Parameter Value[channel definition]	Parameter Value[objective lens]	Parameter Value[numerical aperture]	Raw Data File	Parameter Value[PixelsizeX]	Unit	Term Source REF	Term Accession Number	Parameter Value[PixelsizeY]	Unit	Term Source REF	Term Accession Number	Parameter Value[PixelsizeZ]	Unit	Term Source REF	Term Accession Number	Protocol REF	Data Transformation Name	Derived Data File

```

Comment:
@gerritsmedia It seems that this ISA Assay File is missing an "Assay Name" (and values) right before the "Raw Data File" Field. The Field is required to pass ISA syntactic validation.

Reply:
I checked it with the Isacreator and indeed it seems that I deleted this particular column. I thought, the assay name is already in the filename itself, why repeat it? But, I can add it again.

Reply:
That column refers to the specific assay execution/application, while the assay filename is the name of the assay table, containing the details of all the specific assay application. Could you please to add the column back in for this PR?

Reply:
Some "Assay Name" may have multiple output files hence the field. in your case, it seems indeed somehow duplicating information but it is necessary for it to be present.

Reply:
Thanks @gerritsmedia - feel free to update your repository and the pull request. There are also issues with the protocols and parameter names (e.g. (micro)environment). The ```Protocol REF``` column refers to the protocols defined in the investigation file and the parameters have to match. To link to the documents describing the protocols, it would be better to use the ```Study Protocol URI``` if possible.

Reply:
Some of the syntactic issues: the protocol "Appendix I" was missing the following parameters
"cell culture CO2 partial pressure;cell culture temperature;cell culture antibiotics;cell culture serum concentration;cell culture serum;cell culture medium;cell culture vessel;(micro)environment"

The protocol "Appendix II Microscopy" was defined as "Appendix II Microscopy.pdf" in the investigation file and they need to match in investigation and study files.

Reply:
I will update the files and change the protocol names to *.pdf to make it consistent over the dataset. For the rest, the parameters mentioned above (for appendix I) are all included, however, not with the exact terms. Do I need to include these terms literally? In that case, I need to make some slight adaptations to the protocol in the way it is written. 

What do you mean with (micro)environment? I could adapt the section 'Spheroid embedding in 3D collagen', such that it includes the term '(micro)environment' literally. Could I freely specify the (micro)environment as 'reconstituted fibrillar rattail collagen', or is there a standard term to specify the (micro)environment?

Reply:
Hi, I updated the files, except for the protocol, I will work on that later. Are the isa tab files correct now?

Reply:
The file Appendix I 3D spheroid culture.pdf is also updated now.